docs/changelog.rst

```diff
@@ -89,6 +89,8 @@ Added:
   the changes with ``<return>`` and reject them with ``<escape>``. The
   ``{transformation-info}`` status module displays the currently selected geometry of
   the original image. Thanks `@Yutsuten`_ for reviving this!
+* Add the ``none`` sorting type for the ``sort.image_order`` and ``sort.directory_order``
+  options, implemented by `@buzzingwires`_
```

Comment:
You will also need to define the link at the bottom of the page :wink:

Reply:
Oops! I'm sorry. I haven't worked much with RST files. I'll patch that right away.

Reply:
Thanks! The syntax can be pretty iffy, but sphinx is just so well integrated in python that I guess it's worth it :blush: